chat-plugins/othermetas.js

```diff
@@ -132,6 +132,7 @@ const commands = {
 		}
 		//////////////////////////////////////////
 		let mixedTemplate = Dex.deepClone(template);
+		mixedTemplate.abilities = megaTemplate.abilities;
```

Comment:
This isn't a clone...

Reply:
```suggestion
		mixedTemplate.abilities = Object.assign({}, megaTemplate.abilities);
```

Reply:
I don't know how I didn't do that. But shouldn't this be Dex.deepClone? Or am i missing something from your refactor commit?

Reply:
Good point. In practice they do the same thing, but we should probably use `Dex.deepClone` for consistency.